docs/configuration.md

```diff
@@ -954,16 +970,16 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.memory.offHeap.enabled</code></td>
   <td>false</td>
   <td>
-    If true, Spark will attempt to use off-heap memory for certain operations. If off-heap memory use is enabled, then <code>spark.memory.offHeap.size</code> must be positive.
+    If true, Spark will attempt to use off-heap memory for certain operations(e.g. allocate memory by Unsafe/Tungsten code). If off-heap memory use is enabled, then <code>spark.memory.offHeap.size</code> must be positive.
   </td>
 </tr>
 <tr>
   <td><code>spark.memory.offHeap.size</code></td>
-  <td>0</td>
+  <td>384 * 1024 * 1024</td>
   <td>
     The absolute amount of memory in bytes which can be used for off-heap allocation.
     This setting has no impact on heap memory usage, so if your executors' total memory consumption must fit within some hard limit then be sure to shrink your JVM heap size accordingly.
-    This must be set to a positive value when <code>spark.memory.offHeap.enabled=true</code>.
+    This must be set to a positive value when <code>spark.memory.offHeap.enabled=true</code>. Note that Blocks will be shuffled to off heap by default.
```

Comment:
Yes, I will fix this. I was thinking Netty will use off-heap for fetching remote blocks when `spark.shuffle.io.preferDirectBufs` is true. That's why I put a `by default` here.